testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkIcebergRest.java

```diff
@@ -43,7 +43,7 @@
     private static final int REST_SERVER_PORT = 8181;
     private static final String SPARK_CONTAINER_NAME = "spark";
     private static final String REST_CONTAINER_NAME = "iceberg-with-rest";
-    private static final String REST_SERVER_IMAGE = "tabulario/iceberg-rest:0.2.0";
+    private static final String REST_SERVER_IMAGE = "tabulario/iceberg-rest:0.3.0-snapshot";
```

Comment:
Does Iceberg 1.2 no longer work with previous version of the REST server?
Or is it an independent version bump?

(in any case, we want to depend on exact version, not a snapshot/latest)

Reply:
> (in any case, we want to depend on exact version, not a snapshot/latest)

once Iceberg 1.2.0 is released, that Image will also be bumped to 0.3.0 (which will internally point to a released Iceberg version)